packages/syntax-core/src/SelectList/SelectList.module.css

```diff
@@ -0,0 +1,78 @@
+.selectContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.opacityOverlay {
+  opacity: 0.3;
+}
+
+.outerTextContainer {
+  padding-inline-start: 4px;
+}
+
+.selectWrapper {
+  display: flex;
+  position: relative;
+}
+
+.selectBox {
+  width: 100%;
+  padding: 0 12px;
+  display: flex;
+  align-items: center;
+  border: 1px solid #767676;
```

Comment:
@nathan-ong could we use the `--color-base-gray-700` for this? https://cambly-syntax.vercel.app/?path=/docs/colors--docs